src/qml/CoordinateLocator.qml

```diff
@@ -29,7 +29,13 @@ Item {
 
   // some trickery here: the first part (!mapSettings.visibleExtent) is only there to get a signal when
   // the map canvas extent changes (user pans/zooms) and the calculation of the display position is retriggered
-  readonly property point displayPosition: !!mapSettings.visibleExtent || true ? mapSettings.coordinateToScreen(currentCoordinate) : 0
+  readonly property point displayPosition: {
+    // this property gets initially evaluated before the `currentCoordinate`
```

Comment:
That's surprising. I think the `currentCoordinate` should be acquired through a getter (transparently inside Qt). At this point it will be evaluated, so implicitly when it's required here.

I wonder if

1. my understanding of the process is wrong
2. currentCoordinate is not a bound property (and instead a property that is set/pushed / or depends on a property that is pushed)
3. there's a binding loop

Food for thoughts. This looks good anyway.